pkg/virt-handler/node-labeller/node_labeller_test.go

```diff
@@ -196,6 +196,18 @@ var _ = Describe("Node-labeller ", func() {
 		))
 	})
 
+	It("should not add cpu tsc labels if the counter is nil", func() {
+		nlController.cpuCounter = nil
+		res := nlController.execute()
+		Expect(res).To(BeTrue())
+
+		node := retrieveNode(kubeClient)
+		Expect(node.Labels).To(SatisfyAll(
+			Not(HaveKey(v1.CPUTimerLabel+"tsc-frequency")),
+			Not(HaveKey(v1.CPUTimerLabel+"tsc-scalable")),
+		))
+	})
+
```

Comment:
very small nit: you can combine this test with the above one using a `DescribeTable` and avoid a bit of code duplication. feel free to ignore though :)

Reply:
Done, thanks for reviewing it so fast.